### DISCUSSION TITLE
The U8g2 is ported to the ST7789 LCD screen

### DISCUSSION THREAD

**Discussion by Shijie-star** [olikraus/u8g2#2127] [Q&A]
Hello, I am a college student and I have a problem in the process of completing a project. Port the U8g2 to the ST7789 LCD screen. But because the LCD model matching U8g2 does not include ST7789, I would like to know which LCD model library you used for transplantation.

**Comment by olikraus** [comment]
U8g2 doesn't support RGB Displays.
